doc/supported-hardware.md

```diff
@@ -0,0 +1,61 @@
+# Supported Hardware
+
+The following SR-IOV capable hardware is supported with sriov-network-operator:
+
+| Model                    | Vendor ID | Device ID |
+| ------------------------ | --------- | --------- |
+| Intel XXV710 Family |  8086 | 158b |
+| Intel E810-CQDA2/2CQDA2 Family | 8086  | 1592 |
+| Intel E810-XXVDA4 Family | 8086  | 1593 |
+| Intel E810-XXVDA2 Family | 8086  | 159b |
+| Mellanox MT27700 Family [ConnectX-4] | 15b3 | 1013 |
+| Mellanox MT27710 Family [ConnectX-4 Lx] | 15b3 | 1015 |
+| Mellanox MT27800 Family [ConnectX-5] | 15b3 | 1017 |
+| Mellanox MT28800 Family [ConnectX-5 Ex] | 15b3 | 1019 |
+| Mellanox MT28908 Family [ConnectX-6] | 15b3 | 101b |
+| Mellanox MT28908 Family [ConnectX-6 Dx] | 15b3 | 101d |
+| Mellanox MT42822 BlueField-2 integrated ConnectX-6 Dx | 15b3 | a2d6 |
+
+> **Note:** sriov-network-operator maintains a list of supported NICs which it supports.
+> These are stored in supported-nic-ids [configMap](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator/templates/configmap.yaml).
+> The operator uses this list to enforce it only operates on NICs that are supported. For unsupported SR-IOV NICs, that is not guaranteed, but might work as well.
+> To have sriov-network-operator operate on an unsupported NIC, after installing the operator, you have to add the unsupported SR-IOV NICs information to the ConfigMap
+> in following format: `<nic_name>: <vender_id> <pf_device_id> <vf_device_id>`.
+> Then restart the config daemon and operator webhook pods.
+
+# Adding new Hardware
+
+## Initial support
+Vendors (or other parties) interested in adding a new supported hardware to the supported list of devices of sriov-network-operator
+should follow the following procedure:
+
+* Open an Issue requesting to add support for new hardware
+  * Specify which hardware it is (Model, Vendor ID, Device ID)
+* Perform Testing on requested hardware using:
+  * Kubernetes last release
+  * either last release version or master version of sriov-network-operator
+  * Note: We do not have a specifc list of test-cases however you should at a minimum ensure
+      sriov-network-operator is able to properly discover your device, configure SR-IOV and
+      expose them as a kubernetes node resource, and you are able to run workloads consuming
+      those resources.
+* Add information of what was tested to the issue opened
+* Add contact point information to the opened issue so we know who to reach out if issues arise when running sriov-network-operator against the specified hardware.
```

Comment:
This is a good point.
Would the contact point be from vendor or it could be anyone who adds the card?

Reply:
I think it should be from the interested party who requested to add support.